src/index.php

```diff
@@ -1,3 +1,46 @@
 <?php
-echo "welcome at cronbeat";
+
+define('APP_DIR', dirname(__DIR__));
+
+require_once APP_DIR . '/src/classes/database.php';
+require_once APP_DIR . '/src/controllers/BaseController.php';
+require_once APP_DIR . '/src/controllers/SetupController.php';
+require_once APP_DIR . '/src/controllers/LoginController.php';
+require_once APP_DIR . '/src/controllers/DashboardController.php';
+
+use Cronbeat\Database;
+use Cronbeat\Controllers\SetupController;
+use Cronbeat\Controllers\LoginController;
+use Cronbeat\Controllers\DashboardController;
+
+function parseControllerFromUrl() {
+    $uri = $_SERVER['REQUEST_URI'];
+    $uri = trim($uri, '/');
+    $uri = explode('/', $uri);
+    return !empty($uri[0]) ? $uri[0] : 'login';
+}
```

Comment:
@jetbrains-junie cover this with tests (therefore probably move it to some abstract helper class)